scripts/helpers/tizen_gdbserver_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+SDB="$TIZEN_SDK_ROOT/tools/sdb"
+PORT=9999
```

Comment:
Can we make this port configurable? E.g.: debugging two apps at the same time.

Reply:
Fixed, also I've added possibility do choose target device